docs/configuration.md

```diff
@@ -35,8 +35,21 @@ val conf = new SparkConf()
 val sc = new SparkContext(conf)
 {% endhighlight %}
 
-Note that we can have more than 1 thread in local mode, and in cases like spark streaming, we may actually
-require one to prevent any sort of starvation issues.
+Note that we can have more than 1 thread in local mode, and in cases like spark streaming, we may 
```

Comment:
spark streaming -> Spark Streaming. If in fact microseconds are removed, they can be removed in the doc below this.